identity/migration/src/main/java/io/camunda/identity/migration/MigrationRunner.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.migration;
+
+import io.camunda.service.AuthorizationServices;
+import io.camunda.service.UserServices;
+import io.camunda.zeebe.protocol.impl.record.value.authorization.AuthorizationRecord;
+import io.camunda.zeebe.protocol.impl.record.value.user.UserRecord;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+@Profile("identity-migration")
+public class MigrationRunner implements ApplicationRunner {
+
+  final UserServices<UserRecord> userService;
+
+  final AuthorizationServices<AuthorizationRecord> authorizationServices;
+
+  final RestTemplate restTemplate;
+
+  public MigrationRunner(
+      final UserServices<UserRecord> userService,
+      final AuthorizationServices<AuthorizationRecord> authorizationServices) {
+    this.userService = userService;
+    this.authorizationServices = authorizationServices;
+    restTemplate = new RestTemplate();
+  }
+
+  @Override
+  public void run(final ApplicationArguments args) throws Exception {
+    final String command =
+        args.containsOption("command") ? args.getOptionValues("command").getFirst() : "migrate";
+    final String mode =
+        args.containsOption("mode") ? args.getOptionValues("mode").getFirst() : "resume";
+    final String entities =
+        args.containsOption("entities") ? args.getOptionValues("entities").getFirst() : null;
```

Comment:
💭 as discussed, my understanding of the migration app is that it will run in a way that is designed to pick up from where it was during a pod restart and if there is any issue then we will rollback the cluster. 

Having thought more about it I'm not convinced these options will be used so I would prefer if they were removed for now (and we can add them in later on is something like this is needed) as it makes the code simpler and cleaner.